src/containers/PageSizer.js

```diff
@@ -21,7 +22,7 @@ const mapDispatchToProps = dispatch => bindActionCreators({
 const handlers = {
   changePageSize: props => event => {
     const pageSize = event.target.value
-    props.refreshData(props.name, {...props.filter, pageSize: pageSize})
+    props.refreshData(props.name, {...initialState.filter, pageSize: pageSize})
```

Comment:
It looks like this will reset the sorts and any other data saved in the filter which we probably don't want to do.

Reply:
I though that was the point, so we just want to overwrite the page and pageSize?